src/components/Button/index.stories.tsx

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
 import State from 'util/storybook-docs/State';
+import { Story } from '@storybook/react';
 import { StoryWrapper, StoryItem } from 'util/storybook-docs/StoryLayout';
 import { StarFilledIcon, WorldIcon } from 'lib/icons/react';
 import Button, { ButtonProps, THEMES, Theme, SIZES } from '.';
 
-const Template = (args) => <Button label="Button" {...args} />;
+const Template = ({ label = 'Button', ...args }: ButtonProps) => (
+  <Button {...args} label={label} />
+);
 
 const INVERTED_THEMES: Theme[] = ['ghost', 'outlined'];
 
-export const Primary = Template.bind({});
+export const Primary: Story<ButtonProps> = Template.bind({});
```

Comment:
@mwahlmann-cbi This will be needed on your subsequent story conversions (now that strict mode is enabled)